src/reference/modules/reference/pages/logs.adoc

```diff
@@ -0,0 +1,76 @@
+= Logs reference
+
+== Server side
+
+The Rudder log files are all located in the: `/var/log/rudder` directory.
+The directory hierarchy is quite self-explanatory:
+
+----
+root@server:/var/log/rudder# ls -1
+agent-check
+apache2
+compliance
+core
+install
+ldap
+nodeConfigurations
+rudder-pkg
+webapp
+
+----
+
+=== Webapp
+
+All web application logs are located under the `webapp` folder. A new log file is generated everytime the service `rudder-jetty` is restarted.
+If you want to access the current webapp logs, look for the file named `<today's date>.stderrout.log`.
+The `java.log` is used to log the JVM errors.
+
+=== Apache2
+
+The `apache2` folder is mostly used to log inventories and API calls.
+
+=== Reports
+
+Only active when using the `SYSLOG` reporting mode and enabled in the webapp settings, it aggregates all raw
+reports received from the nodes under the file `/var/log/rudder/reports/all.log`.
+
+=== Compliance
+
+The `compliance` folder will agregate by default all non `Success` or `N/A` reports under `/var/log/rudder/compliance/non-compliant-reports.log`
```

Comment:
There is already a detailed explanation of this format in the doc, it would be better to add a link

Reply:
I could not find the doc page where it is referenced. The only place where it is detailed are the training slides.
In the reference doc we only have a documentation about the report format used by Rudder, which for some reasons is different from the on in those log files.